src/components/tl/ucp/tl_ucp_tag.h

```diff
@@ -39,16 +39,19 @@
 #define UCC_TL_UCP_SCOPE_BITS_OFFSET                                        \
     (UCC_TL_UCP_ID_BITS + UCC_TL_UCP_SENDER_BITS + UCC_TL_UCP_SCOPE_ID_BITS)
 
-#define UCC_TL_UCP_SCOPE_ID_BITS_OFFSET (UCC_TL_UCP_ID_BITS + UCC_TL_UCP_SENDER_BITS)
+#define UCC_TL_UCP_SCOPE_ID_BITS_OFFSET (UCC_TL_UCP_ID_BITS + \
+                                         UCC_TL_UCP_SENDER_BITS)
```

Comment:
This is a code formatting only change and should be a separate patch (or dropped).